contracts/README.md

```diff
@@ -38,6 +38,8 @@ Introducing the development contracts in the order they were created.
    that hackatom has.
 10. **virus** is a contract that reproduces itself and does nothing useful
     beyond that, showing how to use instantiate2 from a contract.
+11. **empty** is an empty contract that does nothing. It is used to test that we
+    can store contracts with no entry points.
```

Comment:
Here we preserve the order since those examples tell a story of CosmWasm evolvement over time